contracts/src/StakeTableV2.sol

```diff
@@ -55,9 +55,47 @@ import { SafeTransferLib } from "solmate/utils/SafeTransferLib.sol";
 ///
 /// 7. The `pause` and `unpause` functions are added for emergency control.
 ///
+/// 8. The commission rate for validators can be updated with the `updateCommission` function.
+///
 /// @notice The StakeTableV2 contract ABI is a superset of the original ABI. Consumers of the
 /// contract can use the V2 ABI, even if they would like to maintain backwards compatibility.
 contract StakeTableV2 is StakeTable, PausableUpgradeable, AccessControlUpgradeable {
+    // === Types ===
+
+    /// @notice Struct for tracking validator commission and last increase time
+    struct CommissionTracking {
+        uint16 commission;
+        uint256 lastIncreaseTime;
```

Comment:
a validator may also decrease commission so maybe rename to `lastModified` or `lastChanged`

Reply:
The decreases are not rate limited, and therefore the timestamp isn't tracked and and the amount of the decrease isn't checked.